index.html

```diff
@@ -238,9 +240,13 @@ <h3>Design Goals and Rationale</h3>
           </dd>
           <dt>Extensibility</dt>
           <dd>
-Creating and deploying new proof types is a fairly trivial undertaking
-to ensure that the proof format increases the rate of innovation in the
-digital proof space.
+This specification provides the structure by which new proof types can be defined
+for use with Verifiable Credentials and other constrained document use cases. This 
+promotes innovation as well as the ability to test and validate cryptographic agility 
```

Comment:
curious here - should this be "measured innovation"
innovating with novel crypto when it's "worth it" (e.g. post-quantum schemes) is great, innovating with experimental, shoddy crypto can be really dangerous!

how do we draw the line?

Reply:
I like that phrasing - incorporating